### PR TITLE
Fix missing CohereConnectionError import in Client

### PR DESCRIPTION
**Pull request by sanderland**

### CODE REVIEW

File: cohere/client.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from urllib3 import Retry
 
 import cohere
-from cohere.error import CohereAPIError, CohereError
+from cohere.error import CohereAPIError, CohereConnectionError, CohereError
```
Comment (abdullahkady) [inline cohere/client.py:14]: how was this not captured by CI linters?

**Review by abdullahkady — APPROVED** [review]

File: cohere/client.py (hunk 1 above)
Comment (sanderland) [reply cohere/client.py:14]: They capture unused imports, not missing ones.

Comment (abdullahkady) [reply cohere/client.py:14]: we definitely should configure that! it's not about imports, it's for undefined variable in general:
`undefined name 'CohereConnectionError'Flake8(F821)`

Comment (sanderland) [reply cohere/client.py:14]: we don't have flake8 as a precommit hook though. Adding it with select errors such as this (not overlapping with formatting) seems like a good idea :)

**Pull request merged**